src/client/components/timer/timer.tsx

```diff
@@ -23,6 +23,8 @@ type TimerProps = {
 const Timer: FC<TimerProps> = ({ active = true, duration, targetTime }) => {
   const [painted, setPainted] = useState(false);
 
+  console.log(`Printing TIMER: active=${active}`);
```

Comment:
Did you want to keep this, or was it just for debugging purposes?

Reply:
Right!  Overlooked this!